doc/vim-asterios.txt

```diff
@@ -0,0 +1,226 @@
+              *%&&@@@@@@@@@%(*.................*@%#%%%&&&&&&&&(*****%        ~
+    &&&&&&(,,..................................@@#%%%%%&&&&&&&******&        ~
+    &&&&&&&&&&&&&&&&&&&@@@@@@@@@@@@@@@@@...../@@#%%%%%%&&&&&&&***/**&        ~
+                     .,,,,..**######(@@.....@@#%%%%%%%&&&&&&&**////&         ~
+                      ,,,,,..***###(@@....(@@#%%%%%%&&&&&&&(*/////&,         ~
+                        *,,,,,,***#@@....@@%%%%%%%&&&&&&&(**/////&           ~
+                         ,*,,,,,,,@@...(@@##%%%&&&&&&&#**//////&,            ~
+                            /,,*,&&...@@#%%%%&&&%/****////(/(&               ~
+                               /%&..(@&**********////////(&                  ~
+                               &&..&&**/////////////(%(                      ~
+                              &&//&&%%*                                      ~
+                               (%%%%%                                        ~
+                                                                             ~
+            #     #####  ####### ####### ######  ### #######  #####  ~
+           # #   #     #    #    #       #     #  #  #     # #     # ~
+          #   #  #          #    #       #     #  #  #     # #       ~
+         #     #  #####     #    #####   ######   #  #     #  #####  ~
+         #######       #    #    #       #   #    #  #     #       # ~
+         #     # #     #    #    #       #    #   #  #     # #     # ~
+         #     #  #####     #    ####### #     # ### #######  #####  ~
+
+==============================================================================
+CONTENTS                                               *vim-asterios-contents*
+
+  1. Overview..........................................|vim-asterios-overview|
+  2. PsyC Linting.....................................|vim-asterios-psyc-lint|
+  3. JSON Completion..................................|vim-asterios-json-comp|
+  4. Configuration.......................................|vim-asterios-config|
+  5. Commands..........................................|vim-asterios-commands|
+  6. Acknowledgments...........................|vim-asterios-acknowledgements|
+
+==============================================================================
+OVERVIEW                                               *vim-asterios-overview*
+
+
+vim-asterios provides support for the languages used when developing an
+ASTERIOS application. It includes out of the box syntax highlighting for:
+
+  * the PsyC (support of the `.psy` files)
+  * the Bgt (support of the `.bgt` files)
+  * the KhiC (support of the `.khic` files) - deprecated
+  * the tak (support of the `.tak` files)   - deprecated
+
+For ASTERIOS K19.2 and higher, vim-asterios also features an integration with
+`ALE` (Asynchronous Lint Engine) to provide linting of PsyC source files, and
+an integration with `coc.nvim` for linting and autocompletion of JSON
```

Comment:
That's really a burden to install nodejs :/

Reply:
Agreed. Unfortunately, there is currently no way around it: the only serious language server for JSON available is the one from Visual Studio Code...